sv_inline.h

```diff
@@ -180,6 +180,7 @@ ALIGNED_TYPE(XPVOBJ);
         STRUCT_OFFSET(type, last_member) \
         + sizeof (((type*)SvANY((const SV *)0))->last_member)
 
+/* sz_if_arena, copy_sz, ghost_sz, SVt, !upg, NV=0; , has_arena, arena_sz */
```

Comment:
I think using the actual member names would be less confusing, even if they don't have the best possible names.